### PR TITLE
Updated lootdrops

### PR DESCRIPTION
**Pull request by Pyko1**
Added the items that we have on live with the same weights.

### CODE REVIEW

**Comment by Pyko1** [comment]
will prolly not compile as some items are still missing.

File: code/game/objects/effects/spawners/lootdrop.dm [hunk 1]
```diff
@@ -97,6 +97,7 @@
 				/obj/item/device/assembly/timer = 3,
 				/obj/item/device/flashlight = 4,
 				/obj/item/device/flashlight/pen = 1,
+				/obj/item/weapon/storage/daki = 3,
```
Comment (Carbonhell) [inline code/game/objects/effects/spawners/lootdrop.dm:100]: err..did you add dakis back?

File: code/game/objects/effects/spawners/lootdrop.dm [hunk 2]
```diff
@@ -107,6 +108,9 @@
 				/obj/item/stack/rods{amount = 10} = 9,
 				/obj/item/stack/rods{amount = 23} = 1,
 				/obj/item/stack/rods{amount = 50} = 1,
+				/obj/item/stack/ducttape{amount = 5} = 6,
+				/obj/item/stack/ducttape{amount = 10} = 2,
```
Comment (Carbonhell) [inline code/game/objects/effects/spawners/lootdrop.dm:112]: same for duct tape?

File: code/game/objects/effects/spawners/lootdrop.dm [hunk 3]
```diff
@@ -156,6 +160,7 @@
 				/obj/item/clothing/gloves/color/random = 8,
 				/obj/item/clothing/shoes/laceup = 1,
 				/obj/item/weapon/storage/secure/briefcase = 3,
+				/obj/item/weapon/relic = 3,
```
Comment (Carbonhell) [inline code/game/objects/effects/spawners/lootdrop.dm:163]: and relics??i actually think we should leave relics out,tg nullrodbased relics are better

**Comment by Pyko1** [comment]
havent added any of em, and prolly wont, its really DNM as it is now, waiting for shit to be added.

File: code/game/objects/effects/spawners/lootdrop.dm (hunk 3 above)
Comment (ArcLumin) [reply code/game/objects/effects/spawners/lootdrop.dm:163]: Relics are the experimentor items lad. And I fuckin love em

**Comment by JohnGinnane** [comment]
Closing due to conflicts and I believe loot spawners were fixed by @TDcoolguy300

**Pull request closed without merging**